python/lsst/dax/obscore_generator/gen_img_obscore.py

```diff
@@ -103,7 +100,7 @@ def exec_gen_obscore(ds_types, out_dir):
             exp = None
             try:
                 wcs = _butler.get(dsType + ".wcs", ref.dataId)
```

Comment:
Is this code still used? I hope not. Am I being confused by the old code not being deleted?

Reply:
Sorry I did not mention this old tool specifically. It is not used in a sense that we are not going to run it, but it was useful for me as a reference when I implemented a new class. It should go at some point (and `bin/` folder too) but for now I still want to keep it unless you say it definitely should disappear 🙂

Reply:
I definitely think it should be cleared out fairly soon. It's confusing to have it there.